pennylane/ops/op_math/__init__.py

```diff
@@ -30,6 +30,17 @@
     ~prod
     ~s_prod
 
+Functions that act on quantum functions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These functions accept quantum functions (Python functions
+containing quantum operations) that are used to construct QNodes.
+
+.. autosummary::
```

Comment:
Does it still make sense to separate it from adjoint and ctrl?

Reply:
+1 - I'd just stick it into the first list